### ISSUE TITLE
Stay signed in?

### ISSUE THREAD

**Issue by aadog** [tuberboy/imap#1]
Microsoft
JesseusfBrowny@outlook.com
Stay signed in?
Stay signed in so you don't have to sign in again next time.

**Comment by tuberboy** [comment]
> 
> Microsoft
> JesseusfBrowny@outlook.com
> Stay signed in?
> Stay signed in so you don't have to sign in again next time.
> 

do you need any help?

**Comment by aadog** [comment]
> > Microsoft 
> > [JesseusfBrowny@outlook.com](mailto:JesseusfBrowny@outlook.com)
> > 保持登录状态？
> > 保持登录状态，以便您下次无需再次登录。
> 
> 你需要帮助吗？

yes

**Comment by tuberboy** [comment]
> > > Microsoft 
> > > [JesseusfBrowny@outlook.com](mailto:JesseusfBrowny@outlook.com)
> > > 保持登录状态？
> > > 保持登录状态，以便您下次无需再次登录。
> > 
> > 你需要帮助吗？
> 
> yes

Please share in details

**Comment by aadog** [comment]
After I log in using my account and password, he needs me to click whether to save the password. What should I do?

**Comment by aadog** [comment]
![image](https://github.com/user-attachments/assets/e0a4883e-0aa5-40ca-af6d-0f4e58aba495)

**Comment by aadog** [comment]
![image](https://github.com/user-attachments/assets/738fab06-0fc4-4d26-8713-ac0e12140551)

**Comment by aadog** [comment]
AnthonympdHendersonq

**Comment by tuberboy** [comment]
First, I want to say: Do not share any personal or confidential information, such as passwords.

You want to programmatically access Outlook without using a browser. Clicking on browser output links won't work because sessions or cookies cannot be transferred to the browser. Instead, you'll need to make the next API call to confirm the action (e.g., pressing "Yes") and then another API call to access the inbox.

Currently, with the existing code, you can successfully log in and retrieve session or cookie data for inbox checks or other tasks using API calls. However, implementing the next API call will require more time. I’ll work on it when I need it or have time to do so.

If you have a specific use case, please share the details, and I’ll check it and implement on existing codes. Thanks

**Comment by aadog** [comment]
Thank you for your reply. I would like to know what app you use to capture the data.

**Comment by tuberboy** [comment]
> Thank you for your reply. I would like to know what app you use to capture the data.

Mitmproxy and Fiddler.

**Comment by aadog** [comment]
> > 谢谢你的回复。我想知道你用什么应用程序来捕获数据。
> 
> Mitmproxy 和 Fiddler。
No, I want to ask, what is the program you use to capture the data, the computer version of Outlook?

**Comment by aadog** [comment]
If I pay can you make an example for me?

**Comment by tuberboy** [comment]
> > > 谢谢你的回复。我想知道你用什么应用程序来捕获数据。
> > 
> > Mitmproxy 和 Fiddler。
> No, I want to ask, what is the program you use to capture the data, the computer version of Outlook?
> 

I used `Android Chrome App` and `Mitmproxy` windows software to capture. You can use any devices or browser to do that, like windows chrome/opera/explorer/firefox, android chrome/opera/firefox, ios safari/chrome/mozilla etc etc all.

**Comment by tuberboy** [comment]
> If I pay can you make an example for me?

You haven't to pay for that. Just clarify your questions what you exactly wanted to do and still wanting, then i can help you.

If you wanting to learn or don't know how to capture traffics from android/windows apps using mitmproxy then watch tutorials on YouTube, you can be learn it easily.

**Comment by aadog** [comment]
> > > > 谢谢你的回复。我想知道你用什么应用程序来捕获数据。
> > > 
> > > 
> > > Mitmproxy 和 Fiddler。
> > > No, I want to ask, what is the program you use to capture the data, the computer version of Outlook?
> 
> I used `Android Chrome App` and `Mitmproxy` windows software to capture. You can use any devices or browser to do that, like windows chrome/opera/explorer/firefox, android chrome/opera/firefox, ios safari/chrome/mozilla etc etc all.

https://outlook.com/？I see your customer ID is rather special

**Comment by aadog** [comment]
> > If I pay can you make an example for me?
> 
> You haven't to pay for that. Just clarify your questions what you exactly wanted to do and still wanting, then i can help you.
> 
> If you wanting to learn or don't know how to capture traffics from android/windows apps using mitmproxy then watch tutorials on YouTube, you can be learn it easily.

I hope there is a complete example, using the account and password to log in to the oauth mailbox, <Skip the password record and oauth authorization> page

**Comment by aadog** [comment]
I am a reverse engineer and I can analyze data packets. I just don’t know what client you are using to capture the data. Outlook? Or Android client? Or Apple client?

**Comment by tuberboy** [comment]
Use web version url, outlook.com then click sign in and then fill forms (email, password) and then check cpatured data for api call.

I will upload a video on here https://youtube.com/tuberboy

wait for it. closing this issue. tnx
